src/components/Search/TimeRangeDropdown.tsx

```diff
@@ -43,6 +56,16 @@ function TimeRangeDropdown({
   const menuItemStyles =
     mode === 'light' ? DropDownItems.Light : DropDownItems.Dark;
 
+  const displayDateRange = (value: number) => {
+    const options: Intl.DateTimeFormatOptions = {
+      dateStyle: 'medium',
+    };
+    const dateFormatter = new Intl.DateTimeFormat('en-US', options);
```

Comment:
Because you'll have to use the same date formatter in SearchOverTime (see above), it would be a good idea to move the format part to another file. I'd suggest a new file `src/utils/format.js`.

In it you could have:
```js
const DATE_FORMATTER = new Intl.DateTimeFormat('en-US', { dateStyle: 'medium' });

export function formatDate(date: Date) {
  return dateFormatter.format(date);
}

export function formatDateRange(date1: Date, date2: Date) {
  return formatDate(date1) + " - " + formatDate(date2);
}
```

And then from `TimeRangeDropdown` you could use `formatDateRange` and just add the parentheses and the extra space.

In the future `formatDateRange` could be improved to not display useless things (maybe), for example when the same year we wouldn't display it. But don't bother with that now.

Reply:
Great idea!

 I went ahead and moved the formatting logic into `src/utils/format.ts` as suggested. This will definitely help keep things clean and reusable. 
 
 Thanks for the feedback! :+1: